x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearchFeatures.java

```diff
@@ -14,8 +14,17 @@
 import org.elasticsearch.xpack.application.connector.ConnectorTemplateRegistry;
 
 import java.util.Map;
+import java.util.Set;
 
 public class EnterpriseSearchFeatures implements FeatureSpecification {
+
+    public static final NodeFeature QUERY_RULES_TEST_API = new NodeFeature("query_rules.test");
```

Comment:
Since the tester isn't context aware, I'm not sure what the best practice is to enforce this feature?